backend/src/gpml/constants.clj

```diff
@@ -7,4 +7,6 @@
    (sort
     (apply conj resource-types ["event" "technology" "policy" "project" "stakeholder" "organisation"]))))
 
+(def reviewer-review-status ["PENDING" "ACCEPTED" "REJECTED"])
```

Comment:
perhaps we can start to use the clojure way for enums  https://stackoverflow.com/a/11856228